etc/docker-entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+set -e
+
+install -d /kaleidoscope/hardware/keyboardio \
+           /kaleidoscope/hardware/keyboardio/avr/libraries/Kaleidoscope \
+           /kaleidoscope-persist/temp  \
+           /kaleidoscope-persist/ccache/cache
+
+echo "Syncing the bundle..."
+tar xf /kaleidoscope-src/bundle.tar -C /kaleidoscope/hardware/keyboardio
+
+echo "Syncing Kaleidoscope..."
+tar xf /kaleidoscope-src/Kaleidoscope.tar -C /kaleidoscope/hardware/keyboardio/avr/libraries/Kaleidoscope
```

Comment:
We're trying to untar `Kaleidoscope.tar` here, while above, we created `kaleidoscope.tar`. On filesystems that are case sensitive, this fails.

Reply:
oops. good catch.